module/applications/sheets/hero-sheet.mjs

```diff
@@ -159,6 +160,28 @@ export default class HeroSheet extends CrucibleBaseActorSheet {
     await this.actor._viewDetailItem("background", {editable: false});
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Handle click actions to edit size bonuses
+   * @this {HeroSheet}
+   * @returns {Promise<void>}
+   */
+  static async #onEditSize() {
```

Comment:
Instead of duplicating this handling on both sheets, I think we should put it on the `CrucibleBaseActorSheet`.